src/compiler/types.ts

```diff
@@ -1646,6 +1646,7 @@ module ts {
         equals = 0x3D,                // =
         exclamation = 0x21,           // !
         greaterThan = 0x3E,           // >
+        hash = 0x23, 

```

Comment:
Comment with the character - older crowds call this a "pound". Crazy, I know. :wink: